stock_quant_manual_assign/wizard/assign_manual_quants.py

```diff
@@ -56,6 +56,9 @@ def assign_quants(self):
             ml.qty_done = ml.product_qty
         move._recompute_state()
         move.mapped('picking_id')._compute_state()
+        # after confirm, the data is useless, so clear the useless data
+        self.quants_lines.unlink()
+        self.unlink()
```

Comment:
Is there an example of a situation where these are needed on transient models?

Reply:
@yostashiro 
yeah, my false, I turn off the auto vacuum by mistake. And I did not know how the transient models works.
Thanks
I think this pull request must be close.